templates/taxi/car_detail.html

```diff
@@ -19,4 +19,12 @@ <h1>Drivers</h1>
       <li>{{ driver.username }} ({{ driver.first_name }} {{ driver.last_name }})</li>
     {% endfor %}
   </ul>
+  <form method="post">
+    {% csrf_token %}
+    {% if user in object.drivers.all %}
```

Comment:
you use here many to many relationship but did you fetched it from db? because otherwise here will be N+1 issue